packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -724,30 +720,168 @@ class TimeInput extends React.Component {
       { 'initial-focus': this.state.secondInitialFocused },
     ]);
 
-    const formatDescriptionId = `terra-time-input-description-format-${this.uuid}`;
-
-    const format = showSeconds
-      ? `(${intl.formatMessage({ id: 'Terra.timeInput.hh' })}:${intl.formatMessage({ id: 'Terra.timeInput.mm' })}:${intl.formatMessage({ id: 'Terra.timeInput.ss' })})`
-      : `(${intl.formatMessage({ id: 'Terra.timeInput.hh' })}:${intl.formatMessage({ id: 'Terra.timeInput.mm' })})`;
+    function mask() {
+      if (showSeconds) {
+        return intl.formatMessage({
+          id: 'Terra.timeInput.maskHourMinute',
+          defaultMessage: '(hh:mm:ss)',
+          description: 'A visual hint that the Time Input expects two-digit hour, minute and second.',
+        });
+      }
+      return intl.formatMessage({
+        id: 'Terra.timeInput.maskHourMinuteSecond',
+        defaultMessage: '(hh:mm)',
+        description: 'Like Terra.timeInput.maskHourMinute but no second.',
+      });
+    }
+
+    /**
+     * Returns a localized description for the hour input that reflects whether this is a 12 or 24-hour clock.
+     *
+     * NOTE: the description does not take the place of validation or error messages.
+     */
+    function hourDescription() {
+      if (is12Hour) {
+        return intl.formatMessage({
+          id: 'Terra.timeInput.hourDescriptionTwelve',
+          defaultMessage: 'A two-digit 12-hour value',
+          description: `Explains to screen reader users that the hour field needs a two digit hour. This will be read
+          only when a screen reader is enabled. It is never displayed. It will be read when the user has focused on
+          the hour input or when the screen reader is reading the page to the user.`,
```

Comment:
Is the use of backticks (`) intentional? this will also preserve the whitespaces so the the second two lines will be tabbed when printed, regardless of indentation of the first line.

Reply:
Yes, it's intentional so I can have this multi-line phrase easily wrapped to 120 chars - my personal max width. Do we have a stance on max width like 80 chars or 120 chars?

I'll see about removing the extra indentation.

Reply:
These descriptions will only appear in our code and (one day, when we upgrade react-intl and update our CI/CD) maybe in our translation team's tools. End users will never see these descriptions.

Reply:
In that case, you can try this. Though if it's only on the dev side then ig this is more of an optional nit.
```suggestion
          description: 'Explains to screen reader users that the hour field needs a two digit hour. This will be read' +
                       'only when a screen reader is enabled. It is never displayed. It will be read when the user has focused on' +
                       'the hour input or when the screen reader is reading the page to the user.',
```

Reply:
Here's a more accurate visualization of the result:
<img width="427" alt="CleanShot 2022-01-07 at 10 54 44@2x" src="https://user-images.githubusercontent.com/38024451/148578433-b515c5b4-4dd2-4681-9077-2c61d24fcc63.png">

Reply:
I'm going to keep with my back ticks for now, because it's the easiest way to reflow the text if it is edited - no need to chop the text into new strings.